### ISSUE TITLE
Possible translation

### ISSUE THREAD

**Issue by karantras** [Night-Pryanik/No-Hope#1]
Is  it possible to add russian translation to this mod?

**Comment by Night-Pryanik** [comment]
Yeah, was thinking about it just yesterday. Will try to translate it and upload new version today.

**Comment by Night-Pryanik** [comment]
Partly translated the mod - mostly as a proof of concept. Try it out.
The translation work is big, so I guess it would take a while to translate it fully.

**Comment by karantras** [comment]
And if you want, I can help you with translation. I have very primitive utility for this goal.

**Comment by Night-Pryanik** [comment]
Yeah, sure, why not?

**Comment by karantras** [comment]
So, how we can communicate? Do you have discord or telegramm?

**Comment by Night-Pryanik** [comment]
Наверное, можно и по-русски общаться, раз мы ведём речь о переводе на русский.

У меня нет ни дискорда, ни телеграмма. 
Самый лучший вариант - запилить пиар с переводами. Тогда я смогу его максимально быстро просмотреть и добавить.
Но можно и просто любым удобным образом переслать мне файл с переведёнными текстами. Хоть сюда, на гитхаб, скинуть.

**Comment by karantras** [comment]
Не совсем понятно, что  именно за пиар. 
Я могу скидывать сначала файлы, где будет чисто переведенный текст, а не в формате json. Если перевод будет устраивать, то уже можно будет засовывать строки в сами файлы. 
И да, я не сильно разбираюсь в том, какие именно строки в json надо переводить, а какие не трогать. И какие файлы  требуют перевода, а какие нет. Сейчас работаю с папкой Ammo. Там два файла. Просто патроны и их тип. Нужно ли переводить тип.

**Comment by Night-Pryanik** [comment]
Нет, переводить нужно только поля name и description (где применимо).
Хорошо, можно скидывать сюда текст, я тогда сам вставлю его в соответствующие файлы.

**Comment by karantras** [comment]
А такие, как "monster_message"? Разве этот текст не пойдет в игру?

**Comment by Night-Pryanik** [comment]
А, ну да, про это я забыл.

**Comment by karantras** [comment]
Тогда пока могу выкладывать переведенные строки здесь:
https://drive.google.com/drive/folders/1v4-X6orsWKRvUyGYJEfEaU3fD8SCXO6G?usp=sharing
Там вопрос про effects. Нужно ли в таких случаях переводить name. Если перевод будет устраивать, то может там через таблицу отмечать это. Чуть позже, я могу присылать непосредственно измененные json, когда допишу программу.

**Comment by Night-Pryanik** [comment]
Я отправил запрос на возможность просмотра таблицы.
